exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -103,19 +106,26 @@ func newPRWExporter(cfg *Config, set exporter.Settings) (*prwExporter, error) {
 		return nil, err
 	}
 
+	err = config.RemoteWriteProtoMsg.Validate(cfg.RemoteWriteProtoMsg)
```

Comment:
It doesn't because we have v1 set in the default config https://github.com/jmichalek132/opentelemetry-collector-contrib/blob/4f04e82994bb865c8d4cf5e649d0e21311e0aaa7/exporter/prometheusremotewriteexporter/factory.go#L102

Reply:
Sure I can add a test case for it if you think it's needed.

Reply:
Should I add the test case or can I mark this as resolved?

Reply:
resolving as agreed on slack.